server/models/prices.py

```diff
@@ -0,0 +1,16 @@
+from database import db
+import datetime
+
+class Prices(db.Model):
+
+    __tablename__ = 'price_history'
+
+    id = db.Column(db.Integer, primary_key=True, unique=True, autoincrement=True)
```

Comment:
we don't need a primary key on this table, it doesn't really do much for us

Reply:
SQLAlchemy doesn't seem to work with tables that don't contain primary keys.

Would it be best instead differentiate by having a `primary_key` and a `product_key`?

The primary key would refer to the individual price entry and the product key the single product.

Reply:
yup, every table in a SQL-like database needs a primary key. It's the way to uniquely identify each row. Then product_key would be our foreign key, connecting this table to the product table.